third_party/move/testing-infra/transactional-test-runner/src/vm_test_harness.rs

```diff
@@ -599,6 +653,7 @@ impl TestRunConfig {
             use_masm: true,
             echo: true,
             cross_compilation_targets: BTreeSet::new(),
+            tracing: false,
```

Comment:
What's the testing strategy (above) to when we set this to true?

Reply:
if set to true, we run paranoid mode via tracing, so now we have 
- all tests run in-place
- tracing only for tracing unit tests
- paranoid tests run both